compiler/src/dotty/tools/dotc/Compiler.scala

```diff
@@ -44,13 +44,13 @@ class Compiler {
       List(new FrontEnd),           // Compiler frontend: scanner, parser, namer, typer
       List(new sbt.ExtractDependencies), // Sends information on classes' dependencies to sbt via callbacks
       List(new PostTyper),          // Additional checks and cleanups after type checking
+      List(new RefChecks,           // Various checks mostly related to abstract members and overriding
```

Comment:
@felixmulder Thanks for looking at it. I tend to agree moving RefChecks before Pickler is a huge change we should try our best to avoid. Let's wait for some better fix that deprecate this one.